internal/workflow/processing_test.go

```diff
@@ -316,6 +317,7 @@ func (s *ProcessingWorkflowTestSuite) TestConfirmation() {
 			Status:     enums.WorkflowStatusInProgress,
 			StartedAt:  startTime,
 			SIPID:      1,
+			SIPUUID:    sipUUID,
```

Comment:
It seems redundant and potentially contradictory to pass SIPID and SIPUUID here. :confused:

Reply:
Fixed in the latest commit.